picard/tagger.py

```diff
@@ -512,80 +545,117 @@ def add_files(self, filenames, target=None):
             log.debug("Adding files %r", new_files)
             new_files.sort(key=lambda x: x.filename)
             if target is None or target is self.unclustered_files:
```

Comment:
As this condition is only setting `target = None` now the condition can be simplified to just `if target is self.unclustered_files:`. Also let's remove the now commented line.

Reply:
Done.